query/src/sql/planner/binder/select.rs

```diff
@@ -55,11 +60,52 @@ impl Binder {
         // Output of current `SELECT` statement.
         let mut output_context = self.normalize_select_list(&stmt.select_list, &input_context)?;
 
+        self.analyze_aggregate(&output_context, &mut input_context)?;
+
+        if !stmt.group_by.is_empty() {
+            self.bind_group_by(&stmt.group_by, &mut input_context)?;
+            output_context.expression = input_context.expression.clone();
+        }
+
         self.bind_projection(&mut output_context)?;
 
         Ok(output_context)
     }
 
+    fn analyze_aggregate(
+        &self,
+        output_context: &BindContext,
+        input_context: &mut BindContext,
+    ) -> Result<()> {
+        let mut agg_expr: Vec<ScalarExprRef> = Vec::new();
+        for agg_scalar in find_aggregate_scalars_from_bind_context(output_context)? {
+            match agg_scalar {
+                Scalar::AggregateFunction {
+                    func_name,
+                    distinct,
+                    params,
+                    args,
+                    data_type,
+                    nullable,
+                } => agg_expr.push(Arc::new(Scalar::AggregateFunction {
+                    func_name,
+                    distinct,
+                    params,
+                    args,
+                    data_type,
+                    nullable,
+                })),
+                _ => {
```

Comment:
What's happen with `SELECT SUM(column_a + 1) FROM table`.

Reply:
`Literal` isn't supported currently ~

Reply:
Or `SELECT SUM(column_a + column_b) FROM table`?

Reply:
Currently, binary operator only supports `Eq`.  Will add more scalar exprs support in the next PRs.